command/repo.go

```diff
@@ -189,7 +189,11 @@ func repoCreate(cmd *cobra.Command, args []string) error {
 	if len(args) > 0 {
 		name = args[0]
 		if strings.Contains(name, "/") {
-			newRepo := ghrepo.FromFullName(name)
+			var err error
```

Comment:
Nit: there is no need to declare `err` here. It will be initialized in the next line due to `:=`

Reply:
doh, yeah good point 😄 a192e85..e598809